Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Xamarin.Forms.Controls.Issues.Shared.projitems

```diff
@@ -1252,6 +1253,9 @@
     <Compile Update="$(MSBuildThisFileDirectory)Issue6130.xaml.cs">
       <DependentUpon>Issue6130.xaml</DependentUpon>
     </Compile>
+    <Compile Update="C:\Users\jfversluis\Documents\GitHub\Xamarin.Forms\Xamarin.Forms.Controls.Issues\Xamarin.Forms.Controls.Issues.Shared\Issue6254.xaml.cs">
```

Comment:
wrong path
```suggestion
    <Compile Update="$(MSBuildThisFileDirectory)Issue6254.xaml.cs">
```

Reply:
Hm he must've come up with that automatically. Need to keep my eye on that to prevent mistakes like these.